README.md

```diff
@@ -4,4 +4,61 @@
 This repository contains various tools that are useful when running pipelines
 with the [Google Genomics API][1]. 
 
+# Building the tools
+
+In order to build the tools, you will need the [Go][2] tool
+chain (at least version 1.8).
+
+Once Go is installed, you can build the tools by running:
+
+```
+$ go get github.com/googlegenomics/pipelines-tools/...
+```
+
+This will produce binaries in $GOPATH/bin for each tool.
+
+# Usage
+
+## The `pipelines` tool
+
+This tool provides support for running, cancelling and inspecting pipelines.
+
+As a simple example, to run a pipeline that prints 'hello world':
+
+```
+$ cat <<EOF > hello.script
+echo "hello world"
+EOF
+$ pipelines --project=my-project run --script=hello.script --output=gs://my-bucket/logs --wait
+```
+
+After the pipeline finishes, you can inspect the output using `gsutil`:
+
+```
+$ gsutil cat gs://my-bucket/logs/output
+```
+
+The script file format is described in the [source code for the command][3].
+
+## The `migrate-pipeline` tool
+
+This tool takes a v1alpha2 run pipeline request and attempts to emit a v2alpha1
```

Comment:
I'd mention that it takes a JSON-formatted v1alpha2 pipelines request.

Reply:
Done